### ISSUE TITLE
[Preview 4] HighContrastChanged crashes app

### ISSUE THREAD

**Issue by asklar** [microsoft/microsoft-ui-xaml#4163]
Found while upgrading react native windows to consume WinUI 3 Preview 4
We call `Windows.UI.ViewManagement.AccessibilitySettings.HighContrastChanged`. This API worked in Preview 3 in both UWP and Desktop, but in Preview 4 this API crashes in Desktop apps.
This worked in previous previews because XAML was calling an unsupported API (to create an invisible CoreWindow) and as a side effect some system APIs like this one worked

Could we update the relnotes @anawishnoff @ryandemopoulos regarding the HighContrastChanged event and other APIs/Events that won't work? I see there is a partial list in the notes already but this one seems like it fell through the cracks.
https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#known-issues
Are there others that might be missing from the relnotes?

**Comment by anawishnoff** [comment]
Thanks! I've made a request to update the release notes, and will continue to keep an eye out for other APIs that may have become unsupported.

@JesseCol for FYI.

**Comment by citelao** [comment]
@JesseCol @asklar, what's the workaround here? How can apps detect if high contrast is enabled? As far as I can tell, we'd have to listen to WM_SETTINGCHANGE and read SPI manually, I think.

**Comment by JesseCol** [comment]
Hi, does this **ThemeSettings.Changed** API work for you?

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.system.themesettings.changed?view=windows-app-sdk-1.4

**Comment by adstep** [comment]
This still repros for me on 1.5.240802000.

**Comment by JesseCol** [comment]
Hi, did you try the ThemeSettings.Changed event instead?  https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.system.themesettings.changed?view=windows-app-sdk-1.4

**Comment by adstep** [comment]
Got it working with ThemeSettings.

**Comment by citelao** [comment]
Working to document this change:

[UISettings PR](https://github.com/MicrosoftDocs/winrt-api/pull/2491)
[API mapping PR](https://github.com/MicrosoftDocs/windows-dev-docs/pull/5300)

**Comment by scfeng10** [comment]
The [ThemeSettings API](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.system.themesettings.changed?view=windows-app-sdk-1.4) is the correct API to use to listen to theme changes, but the Changed event does not fire when switching from a HighContrastBlack to a HighContrastBlack theme or when switching from a HighContrastWhite to a HighContrastWhite theme
